spec/services/deposits_okay_service_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+describe Umrdr::DepositsOkayService do
+  okay = double(Umrdr::DepositsOkayService)
+
+  it "can not find a uniqname known to have no entry" do
+    allow(okay).to receive(new("zzzzzz")).and_return(false)
+    expect(okay.deposit_okay_for("zzzzzz")).to be_falsey
```

Comment:
This test is cannot fail regardless of the definition of DepositoOkayService.  Throw a binding.pry in here after your allow statement and check `okay.class` and the value of `okay`.